app/models/manageiq/providers/dummy_provider/inventory/persister/cloud_manager.rb

```diff
@@ -1,5 +1,16 @@
-class ManageIQ::Providers::DummyProvider::Inventory::Persister::CloudManager < ManagerRefresh::Inventory::Persister
-  include ManagerRefresh::Inventory::CloudManager
+class ManageIQ::Providers::DummyProvider::Inventory::Persister::CloudManager < ManageIQ::Providers::DummyProvider::Inventory::Persister
+  def initialize_inventory_collections
+    add_collection(cloud, :vms)
+    add_collection(cloud, :flavors)
+  end
 
-  has_cloud_manager_vms
+  def parent
+    manager.presence
+  end
+
+  def shared_options
+    {
+      :parent => parent
```

Comment:
@slemrmartin it seems like without this https://github.com/ManageIQ/manageiq/blob/master/app/models/manager_refresh/save_collection/saver/base.rb#L12 will blow up here https://github.com/ManageIQ/manageiq/blob/master/app/models/manager_refresh/inventory_collection.rb#L964
Is there any reason this isn't in core?

Reply:
@agrare previously I added missing `parent ` in `add_collection`.
We can add it to core to `shared_options`, no problem.

Reply:
:+1: awesome thanks @slemrmartin